CHANGELOG.md

```diff
@@ -19,52 +19,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-17
 
+Amazon Chime SDK React Components Library v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK React Components Library v3 includes major improvements for component style customization, `MeetingProvider/MeetingManager` configuration, device management, WebRTC metrics and logger.
+
+- **Component style customization**: Improve components for easier customization of style. Publish this [styling guide](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/styling-guide--page) to instruct on how to customize the component style.
+- **`MeetingProvider/MeetingManager` configuration**: Remove most of `Props` of `MeetingProvider` and improve `MeetingManager.join()` API to allow configuring `MeetingSession` and some other options (e.g., `enableWebAudio`, `logger`) before joining a meeting instead of when `MeetingProvider` is mounted.
```

Comment:
Suggest below statement:
Improve `MeetingProvider`, `MeetingManager` and `MeetingManager.join()` to accept necessary parameters over any redundant parameters. 

We list the changes in detail in migration so just above line should be fine.

Reply:
I think it's worth mentioning "allow configuring before joining a meeting instead of when `MeetingProvider` is mounted."

Updated accordingly.